### ISSUE TITLE
Wrong page offered

### ISSUE THREAD

**Issue by mirh** [internetarchive/FirefoxNoMore404s#53]
Open [this](http://www.madshrimps.be/?action=getarticle&number=6&artpage=2763&articID=604). 
Get the FirefoxNoMore404s bar. 
Notice how the website presented there isn't the right one.

**Comment by rugk** [comment]
Yeah, this issue seems to be similar to https://github.com/internetarchive/FirefoxNoMore404s/issues/54 one.

**Comment by mirh** [comment]
Also with [this](https://appdb.winehq.org/objectManager.php?sClass=version&iId=17421).
